docker-compose.yml

```diff
@@ -150,7 +150,7 @@ services:
     image: node:18-alpine
     environment:
       BLOB_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://blob-storage:10000/devstoreaccount1;QueueEndpoint=http://blob-storage:10001/devstoreaccount1;
-      STORAGE_CONTAINER_NAME: document-service-uploads
+      STORAGE_CONTAINER_NAME: uploads
```

Comment:
In the deployed env the container is called uploads, we thought it would be best to align the local and deployed envs